src/v/config/configuration.cc

```diff
@@ -304,6 +304,13 @@ configuration::configuration()
       "Maximum number of bytes returned in fetch request",
       required::no,
       55_MiB)
+  , metadata_status_wait_timeout_ms(
+      *this,
+      "metadata_status_wait_timeout_ms",
+      "Maximum time to wait in metadata request for cluster health to be "
+      "refreshed",
+      required::no,
+      2s)
```

Comment:
This seems high, for what should be an RPC to an in-memory data structure on the leader (my totally subjective gut feeling would be something like 500ms?), but we can always tune it later.

Reply:
i've checked the default in Kafka client, it is 60 seconds, i think we may leave it 2s for now and tune it in future.